apis/nucleus/src/components/listbox/assets/get-list-sizes.js

```diff
@@ -0,0 +1,58 @@
+const scrollBarWidth = 10; // TODO: ignore this - instead set the styling only show on hover...
+
+export default function getListSizes({ layout, width, height, checkboxes, listCount, count, textWidth }) {
+  const { layoutOptions = {} } = layout || {};
+
+  const { layoutOrder, maxVisibleRows = {}, maxVisibleColumns, dense } = layoutOptions || {};
```

Comment:
```suggestion
  const { layoutOrder, maxVisibleRows = {}, maxVisibleColumns, dense } = layoutOptions;
```
I think it should be fine like this